### PR TITLE
#701 add containers docs

### PR DESCRIPTION
**Pull request by chia7712**
### Checklist
- [x] [Review Contribution Guidelines](https://github.com/oharastream/ohara/blob/master/CONTRIBUTING.md)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#authors))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x]  DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #701)
- [x] Add unit tests (or please explain why this PR is not worth having new tests)
see [link](https://github.com/chia7712/ohara/blob/fix_701_6/docs/rest_interface.md)
- [x] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: docs/rest_interface.md [hunk 1]
```diff
@@ -2614,6 +2625,85 @@ for you. The element **setting** is what you request to validate.
 1. value (**string**) — the value you request to validate
 1. errors (**array(string)**) — error message when the input value is illegal to connector 
 ----------
+## Container
+
+All process managed by ohara is based on docker container. This APIs provide the container details of a running cluster.
```
Comment (saivirtue) [inline docs/rest_interface.md]: a little question, is this implied we will have another doc for k8s container?

Comment (chia7712) [reply docs/rest_interface.md]: Not really. ContainerInfo is a part of interface of collie, so both of them are required to return ContainerInof

Comment (chia7712) [reply docs/rest_interface.md]: It would be better to keep restful APIs consistent even though we have different implementation in the backend.

Comment (saivirtue) [reply docs/rest_interface.md]: got it :)

**Pull request merged**